src/app/shared/animations.ts

```diff
@@ -9,6 +9,13 @@ const fadeInOut = [
     transition("in => out", [animate("0.5s")]),
     // todo - could use :enter and :exit properties also (need to confirm doesn't break existing functionality)
   ]),
+  trigger("noFade", [
+    state("in", style({ opacity: 1 })),
+    state("out", style({ opacity: 0 })),
+    transition("* => in", [
+      animate("0s 0.5s", style({ opacity: 0 })), // delay animation for 1.5s
```

Comment:
You could update this comment and make it a bit more descriptive, e.g.: "delay animation for 0.5s in order to allow previous animation to finish.

Reply:
Added comment.